proto/ibc/core/channel/v1/channel.proto

```diff
@@ -74,15 +74,6 @@ enum State {
   STATE_FLUSHING = 5;
   // A channel has just completed flushing any in-flight packets.
   STATE_FLUSHCOMPLETE = 6;
-  // A channel has just started the channel upgrade handshake.
```

Comment:
best to leave the custom name for new states for another PR yea?

Reply:
until the flush status is removed!